proposals/first-class-span-types.md

```diff
@@ -61,11 +64,21 @@ The explicit span conversions are the following:
 There is no standard explicit span conversion unlike other *standard explicit conversions* ([§10.4.3][standard-explicit-conversions])
 which always exist given the opposite standard implicit conversion.
 
+#### User defined conversions
+
 User-defined conversions are not considered when converting between
 - any single-dimensional `array_type` and `System.Span<T>`/`System.ReadOnlySpan<T>`,
 - any combination of `System.Span<T>`/`System.ReadOnlySpan<T>`,
 - `string` and `System.ReadOnlySpan<char>`.
 
+The implicit span conversions are exempted from the rule
+that it is not possible to define a user-defined operator between types for which a non-user-defined conversion exists
+([§10.5.2 Permitted user-defined conversions][permitted-udcs]).
+This is needed so BCL can keep defining the existing Span conversion operators even when they switch to C# 13
+(to avoid binary breaking changes and also because we use these operators in codegen of the new standard span conversion).
```

Comment:
Perhaps:
```suggestion
(to avoid binary breaking changes and also because these operators are used in codegen of the new standard span conversion).
```